plugins/pulp_python/plugins/models.py

```diff
@@ -1,120 +1,85 @@
-from gettext import gettext as _
 import hashlib
-import re
-import tarfile
 
 from mongoengine import StringField
 from pulp.server.db.model import FileContentUnit
+import twine  # noqa
```

Comment:
That is weird. I feel like someone else had a similar issue with import ordering recently, but I can't recall who.